test/registration/test_registration_api.cpp

```diff
@@ -655,6 +656,55 @@ TEST (PCL, TransformationEstimationPointToPlane)
       EXPECT_NEAR (estimated_transform_double (i, j), ground_truth_tform (i, j), 1e-3);
 }
 
+TEST (PCL, TransformationEstimationSymmetricPointToPlaneLLS)
+{
+  pcl::registration::TransformationEstimationSymmetricPointToPlaneLLS<pcl::PointNormal, pcl::PointNormal> transform_estimator;
+
+  // Create a test cloud
```

Comment:
What is this test case representing physically?

In case the magic numbers need to be updated in the future, some physical intuition can guide the change.

Reply:
Added a comment with the corresponding rotation and translation values.